ccl/backupccl: TestFullClusterBackup failed

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/389e6840-3599-499a-b0e2-9cd00230a383?testReportRun=3&testReportShard=29&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [e2e88aa288c81282b20ca0640da983f97b234324](https://github.com/cockroachdb/cockroach/commits/e2e88aa288c81282b20ca0640da983f97b234324):


```
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestFullClusterBackup1360044476
    test_log_scope.go:81: use -show-logs to present logs inline
    full_cluster_backup_restore_test.go:379: -- test log scope end --
test logs left over in: outputs.zip/logTestFullClusterBackup1360044476
--- FAIL: TestFullClusterBackup (404.47s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (404.33s)
=== RUN   TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored
    full_cluster_backup_restore_test.go:324: query 'SELECT name, "columnIDs", "rowCount" FROM system.table_statistics': expected:
        __auto__, {1}, 2
        __auto__, {4}, 2
        __auto__, {2}, 2
        __auto__, {3}, 2
        __auto__, {1}, 1
        __auto__, {1}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        __auto__, {4}, 1
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {3}, 1
        __auto__, {1}, 58
        __auto__, {2}, 58
        __auto__, {1,2}, 58
        __auto__, {3}, 58
        __auto__, {1,2,3}, 58
        __auto__, {4}, 58
        __auto__, {3}, 1
        __auto__, {1}, 1
        __auto__, {1,3}, 1
        __auto__, {2}, 1
        NULL, {1}, 100
        
        got:
        __auto__, {4}, 2
        __auto__, {3}, 2
        __auto__, {2}, 2
        __auto__, {1}, 2
        __auto__, {1}, 1
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {4}, 1
        __auto__, {3}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (0.90s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=29</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34819

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/8d5b8c67-35b6-4168-965f-87485669da8d?testReportRun=3&testReportShard=29&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [ff291e3ee70eedac50b985860502d2f0b0867003](https://github.com/cockroachdb/cockroach/commits/ff291e3ee70eedac50b985860502d2f0b0867003):


```
=== RUN   TestFullClusterBackup
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestFullClusterBackup866190587
    test_log_scope.go:81: use -show-logs to present logs inline
    full_cluster_backup_restore_test.go:379: -- test log scope end --
--- FAIL: TestFullClusterBackup (496.54s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (496.52s)
=== RUN   TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored
    full_cluster_backup_restore_test.go:324: query 'SELECT name, "columnIDs", "rowCount" FROM system.table_statistics': expected:
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {3}, 0
        __auto__, {1,2,3}, 0
        __auto__, {4}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {3}, 0
        __auto__, {1,2,3}, 0
        __auto__, {4}, 0
        __auto__, {5}, 0
        NULL, {1}, 100
        
        got:
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,2,3}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (3.71s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=29</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/3784f56c-a4ac-49a5-931b-f8831df67029?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [d2d974202781321ddb4fb1115c68fe7566ab9d06](https://github.com/cockroachdb/cockroach/commits/d2d974202781321ddb4fb1115c68fe7566ab9d06):


```
        __auto__, {1}, 58
        __auto__, {2}, 58
        __auto__, {1,2}, 58
        __auto__, {3}, 58
        __auto__, {1,2,3}, 58
        __auto__, {4}, 58
        __auto__, {1}, 30
        __auto__, {2}, 30
        __auto__, {1,2}, 30
        __auto__, {3}, 30
        __auto__, {1,2,3}, 30
        __auto__, {4}, 30
        NULL, {1}, 100
        
        got:
        __auto__, {8}, 0
        __auto__, {1}, 0
        __auto__, {1,8}, 0
        __auto__, {2}, 0
        __auto__, {1,2,8}, 0
        __auto__, {3}, 0
        __auto__, {1,2,3,8}, 0
        __auto__, {4}, 0
        __auto__, {1,2,3,4,8}, 0
        __auto__, {1,3}, 0
        __auto__, {9}, 0
        __auto__, {1,3,9}, 0
        __auto__, {10}, 0
        __auto__, {1,3,10}, 0
        __auto__, {11}, 0
        __auto__, {1,3,11}, 0
        __auto__, {12}, 0
        __auto__, {1,3,12}, 0
        __auto__, {13}, 0
        __auto__, {1,3,13}, 0
        __auto__, {14}, 0
        __auto__, {1,3,14}, 0
        __auto__, {5}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {4}, 1
        __auto__, {3}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        __auto__, {1}, 1
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (0.96s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/9e314f26-e389-4c63-9066-d75bf5b1bb33?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
        __auto__, {1,2,3,4,8}, 0
        __auto__, {1,3}, 0
        __auto__, {9}, 0
        __auto__, {1,3,9}, 0
        __auto__, {10}, 0
        __auto__, {1,3,10}, 0
        __auto__, {11}, 0
        __auto__, {1,3,11}, 0
        __auto__, {12}, 0
        __auto__, {1,3,12}, 0
        __auto__, {13}, 0
        __auto__, {1,3,13}, 0
        __auto__, {14}, 0
        __auto__, {1,3,14}, 0
        __auto__, {5}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {4}, 0
        __auto__, {1,4}, 0
        __auto__, {3}, 0
        __auto__, {6}, 1
        __auto__, {1}, 1
        __auto__, {1,6}, 1
        __auto__, {2}, 1
        __auto__, {3}, 1
        __auto__, {4}, 1
        __auto__, {5}, 1
        __auto__, {7}, 1
        __auto__, {1}, 0
        __auto__, {2}, 0
        NULL, {1}, 100
        
        got:
        __auto__, {4}, 5
        __auto__, {3}, 5
        __auto__, {2}, 5
        __auto__, {1,2}, 5
        __auto__, {1}, 5
        __auto__, {1}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,4}, 0
        __auto__, {1,2}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (1.15s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/05b0e99b-d107-4e85-a3f1-e9a9d88b8559?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
        __auto__, {5}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {3}, 1
        __auto__, {1}, 1
        __auto__, {1,3}, 1
        __auto__, {2}, 1
        __auto__, {6}, 10
        __auto__, {1}, 10
        __auto__, {1,6}, 10
        __auto__, {2}, 10
        __auto__, {3}, 10
        __auto__, {4}, 10
        __auto__, {5}, 10
        __auto__, {7}, 10
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {5}, 0
        __auto__, {2,5}, 0
        __auto__, {3}, 0
        __auto__, {4}, 0
        __auto__, {2}, 1
        __auto__, {1}, 1
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {4}, 1
        __auto__, {3}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        __auto__, {1}, 1
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {4}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,4}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {1}, 1
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (1.95s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (578.94s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/fd2b1628-6e2d-4c90-a5c2-a9a4610deb39?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
=== RUN   TestFullClusterBackup
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestFullClusterBackup1452665621
    test_log_scope.go:81: use -show-logs to present logs inline
    full_cluster_backup_restore_test.go:379: -- test log scope end --
test logs left over in: outputs.zip/logTestFullClusterBackup1452665621
--- FAIL: TestFullClusterBackup (871.35s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (871.33s)
=== RUN   TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored
    full_cluster_backup_restore_test.go:324: query 'SELECT name, "columnIDs", "rowCount" FROM system.table_statistics': expected:
        __auto__, {1}, 1
        __auto__, {2}, 1
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {3}, 0
        __auto__, {1,2,3}, 0
        __auto__, {8}, 0
        __auto__, {1,8}, 0
        __auto__, {9}, 0
        __auto__, {1,9}, 0
        __auto__, {11}, 0
        __auto__, {1,11}, 0
        __auto__, {12}, 0
        __auto__, {1,12}, 0
        __auto__, {13}, 0
        __auto__, {1,13}, 0
        __auto__, {14}, 0
        __auto__, {1,14}, 0
        __auto__, {4}, 0
        __auto__, {5}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {10}, 0
        NULL, {1}, 100
        
        got:
        __auto__, {2}, 1
        __auto__, {1}, 1
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (4.84s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/3c84960c-5927-477f-881f-5c6154b5b97d?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
        __auto__, {15}, 0
        __auto__, {1,15}, 0
        __auto__, {16}, 0
        __auto__, {1,16}, 0
        __auto__, {17}, 0
        __auto__, {1,17}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {8}, 0
        __auto__, {9}, 0
        __auto__, {10}, 0
        __auto__, {13}, 0
        __auto__, {4}, 2
        __auto__, {3}, 2
        __auto__, {2}, 2
        __auto__, {1}, 2
        __auto__, {2}, 1
        __auto__, {1}, 1
        __auto__, {1}, 5
        __auto__, {4}, 5
        __auto__, {3}, 5
        __auto__, {2}, 5
        __auto__, {1,2}, 5
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {1}, 1
        __auto__, {2,5}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {5}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {1}, 0
        __auto__, {7}, 0
        __auto__, {6}, 0
        __auto__, {5}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (2.00s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (489.99s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/3c84960c-5927-477f-881f-5c6154b5b97d?testReportRun=3&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
        __auto__, {2}, 7
        __auto__, {1,2}, 7
        __auto__, {3}, 7
        __auto__, {1,2,3}, 7
        __auto__, {4}, 7
        __auto__, {1,2,3,4}, 7
        __auto__, {5}, 7
        __auto__, {2}, 1
        __auto__, {1}, 1
        __auto__, {4}, 6
        __auto__, {3}, 6
        __auto__, {2}, 6
        __auto__, {1}, 6
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1}, 0
        __auto__, {3}, 5
        __auto__, {2}, 5
        __auto__, {1,2}, 5
        __auto__, {1}, 5
        __auto__, {4}, 5
        __auto__, {1}, 1
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {4}, 1
        __auto__, {3}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,2,3}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1}, 1
        __auto__, {5}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2,5}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (2.04s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/3248e418-de91-40d2-871c-3359f9d01e35?testReportRun=3&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [4f31fbae529e459c00ef2490e193c0c246e8d0f9](https://github.com/cockroachdb/cockroach/commits/4f31fbae529e459c00ef2490e193c0c246e8d0f9):


```
=== RUN   TestFullClusterBackup
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestFullClusterBackup2916361022
    test_log_scope.go:81: use -show-logs to present logs inline
    full_cluster_backup_restore_test.go:379: -- test log scope end --
test logs left over in: outputs.zip/logTestFullClusterBackup2916361022
--- FAIL: TestFullClusterBackup (347.94s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (347.93s)
=== RUN   TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored
    full_cluster_backup_restore_test.go:324: query 'SELECT name, "columnIDs", "rowCount" FROM system.table_statistics': expected:
        __auto__, {1}, 0
        __auto__, {5}, 0
        __auto__, {4}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {2}, 0
        __auto__, {3}, 0
        __auto__, {8}, 0
        __auto__, {9}, 0
        __auto__, {1}, 5
        __auto__, {2}, 5
        __auto__, {1,2}, 5
        __auto__, {3}, 5
        __auto__, {4}, 5
        NULL, {1}, 100
        
        got:
        __auto__, {4}, 5
        __auto__, {3}, 5
        __auto__, {2}, 5
        __auto__, {1,2}, 5
        __auto__, {1}, 5
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (1.25s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/9a7a0fad-4521-4f50-8729-bf6d8f8789cb?testReportRun=3&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [57bd0a92676b1c58ee7e29979ce3dceb5837abdb](https://github.com/cockroachdb/cockroach/commits/57bd0a92676b1c58ee7e29979ce3dceb5837abdb):


```
=== RUN   TestFullClusterBackup
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestFullClusterBackup3808568150
    test_log_scope.go:81: use -show-logs to present logs inline
    full_cluster_backup_restore_test.go:379: -- test log scope end --
test logs left over in: outputs.zip/logTestFullClusterBackup3808568150
--- FAIL: TestFullClusterBackup (404.00s)
=== RUN   TestFullClusterBackup/inSystemTenant=false
    testutils.go:289: no Invalid Descriptors
    testutils.go:289: no Invalid Descriptors
    --- FAIL: TestFullClusterBackup/inSystemTenant=false (404.00s)
=== RUN   TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored
    full_cluster_backup_restore_test.go:324: query 'SELECT name, "columnIDs", "rowCount" FROM system.table_statistics': expected:
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {3}, 0
        __auto__, {1,2,3}, 0
        __auto__, {4}, 0
        __auto__, {5}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {6,7}, 0
        __auto__, {3}, 0
        __auto__, {4}, 0
        __auto__, {5}, 0
        __auto__, {8}, 0
        __auto__, {9}, 0
        __auto__, {10}, 0
        __auto__, {11}, 0
        __auto__, {12}, 0
        __auto__, {13}, 0
        __auto__, {14}, 0
        __auto__, {15}, 0
        __auto__, {16}, 0
        __auto__, {17}, 0
        __auto__, {18}, 0
        __auto__, {19}, 0
        __auto__, {20}, 0
        __auto__, {21}, 0
        __auto__, {22}, 0
        NULL, {1}, 100
        
        got:
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (3.36s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/8c1cff8d-bd76-4a94-ba94-492fc11003b7?testReportRun=2&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
        __auto__, {1,2,3,11}, 0
        __auto__, {4}, 0
        __auto__, {1,2,3,4,11}, 0
        __auto__, {5}, 0
        __auto__, {1,2,3,4,5,11}, 0
        __auto__, {6}, 0
        __auto__, {1,2,3,4,5,6,11}, 0
        __auto__, {2,3}, 0
        __auto__, {1,5}, 0
        __auto__, {14}, 0
        __auto__, {1,5,14}, 0
        __auto__, {15}, 0
        __auto__, {1,5,15}, 0
        __auto__, {16}, 0
        __auto__, {1,5,16}, 0
        __auto__, {17}, 0
        __auto__, {1,5,17}, 0
        __auto__, {18}, 0
        __auto__, {1,5,18}, 0
        __auto__, {19}, 0
        __auto__, {1,5,19}, 0
        __auto__, {7}, 0
        __auto__, {8}, 0
        __auto__, {9}, 0
        __auto__, {10}, 0
        __auto__, {12}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        NULL, {1}, 100
        
        got:
        __auto__, {1}, 0
        __auto__, {5}, 0
        __auto__, {4}, 0
        __auto__, {6}, 0
        __auto__, {7}, 0
        __auto__, {2}, 0
        __auto__, {3}, 0
        __auto__, {8}, 0
        __auto__, {9}, 0
        __auto__, {1}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        __auto__, {4}, 1
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {3}, 1
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (1.25s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/f4a84354-673d-45b5-87e7-b12209e8eb4b?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
        __auto__, {5}, 0
        __auto__, {1,2,3,4,5,11}, 0
        __auto__, {6}, 0
        __auto__, {1,2,3,4,5,6,11}, 0
        __auto__, {2,3}, 0
        __auto__, {1,5}, 0
        __auto__, {14}, 0
        __auto__, {1,5,14}, 0
        __auto__, {15}, 0
        __auto__, {1,5,15}, 0
        __auto__, {16}, 0
        __auto__, {1,5,16}, 0
        __auto__, {17}, 0
        __auto__, {1,5,17}, 0
        __auto__, {18}, 0
        __auto__, {1,5,18}, 0
        __auto__, {19}, 0
        __auto__, {1,5,19}, 0
        __auto__, {7}, 0
        __auto__, {8}, 0
        __auto__, {9}, 0
        __auto__, {10}, 0
        __auto__, {12}, 0
        __auto__, {6}, 10
        __auto__, {1}, 10
        __auto__, {1,6}, 10
        __auto__, {2}, 10
        __auto__, {3}, 10
        __auto__, {4}, 10
        __auto__, {5}, 10
        __auto__, {7}, 10
        __auto__, {2}, 1
        __auto__, {1}, 1
        __auto__, {4}, 6
        __auto__, {3}, 6
        __auto__, {2}, 6
        __auto__, {1}, 6
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1}, 0
        __auto__, {6}, 0
        __auto__, {5}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1}, 0
        __auto__, {7}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (1.08s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/6f2cff4d-c2d3-4903-8f72-e74af552ca31?testReportRun=3&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [c316d6a615fa02a05357a20bf03b8a197ab27810](https://github.com/cockroachdb/cockroach/commits/c316d6a615fa02a05357a20bf03b8a197ab27810):


```
        __auto__, {2,6}, 19
        __auto__, {2,3,6}, 19
        __auto__, {10}, 19
        __auto__, {7}, 19
        __auto__, {8}, 19
        __auto__, {9}, 19
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {3}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {4}, 0
        __auto__, {1,4}, 0
        __auto__, {3}, 0
        __auto__, {1}, 0
        __auto__, {1,6}, 0
        __auto__, {2}, 0
        __auto__, {1,2,6}, 0
        __auto__, {3}, 0
        __auto__, {1,2,3,6}, 0
        __auto__, {4}, 0
        __auto__, {1,2,3,4,6}, 0
        __auto__, {5}, 0
        NULL, {1}, 100
        
        got:
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {3}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {3}, 0
        __auto__, {4}, 0
        __auto__, {5}, 0
        __auto__, {4}, 2
        __auto__, {3}, 2
        __auto__, {2}, 2
        __auto__, {1}, 2
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,4}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (1.51s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestFullClusterBackup [failed](https://tanzanite.cluster.engflow.com/invocations/default/74e807d2-2fb6-494d-9dce-f748f9ec8fd4?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cvY2NsL2JhY2t1cGNjbDpiYWNrdXBjY2xfdGVzdA==) on master @ [435a330ddf15f70234747ef55d3d3721a09ba252](https://github.com/cockroachdb/cockroach/commits/435a330ddf15f70234747ef55d3d3721a09ba252):


```
        __auto__, {1,2,3}, 0
        __auto__, {4}, 0
        __auto__, {5}, 0
        __auto__, {1}, 55
        __auto__, {2}, 55
        __auto__, {1,2}, 55
        __auto__, {3}, 55
        __auto__, {1,2,3}, 55
        __auto__, {4}, 55
        __auto__, {1}, 0
        __auto__, {2}, 0
        __auto__, {1,2}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 1
        __auto__, {1}, 1
        __auto__, {4}, 6
        __auto__, {3}, 6
        __auto__, {2}, 6
        __auto__, {1}, 6
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1}, 0
        __auto__, {3}, 5
        __auto__, {2}, 5
        __auto__, {1,2}, 5
        __auto__, {1}, 5
        __auto__, {4}, 5
        __auto__, {1}, 1
        __auto__, {5}, 1
        __auto__, {4,5}, 1
        __auto__, {4}, 1
        __auto__, {3}, 1
        __auto__, {2}, 1
        __auto__, {1,2}, 1
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        __auto__, {1,2,3}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {1,4}, 0
        __auto__, {1,2}, 0
        __auto__, {1}, 0
        __auto__, {4}, 0
        __auto__, {3}, 0
        __auto__, {2}, 0
        NULL, {1}, 100
        
        --- FAIL: TestFullClusterBackup/inSystemTenant=false/ensure_system_table_data_restored (0.83s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
+++ b/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
@@ -34,6 +34,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog/systemschema"
 	"github.com/cockroachdb/cockroach/pkg/sql/execinfra"
 	"github.com/cockroachdb/cockroach/pkg/sql/execinfrapb"
+	"github.com/cockroachdb/cockroach/pkg/sql/stats"
 	"github.com/cockroachdb/cockroach/pkg/storage"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/jobutils"
@@ -73,6 +74,10 @@ func TestFullClusterBackup(t *testing.T) {
 		}
 
 		settings := clustersettings.MakeTestingClusterSettings()
+		// Disable automatic stats collection on the backup and restoring clusters to ensure
+		// the test is deterministic.
+		stats.AutomaticStatisticsClusterMode.Override(context.Background(), &settings.SV, false)
+
 		params := base.TestClusterArgs{
 			ServerArgs: base.TestServerArgs{
 				Settings:          settings,
@@ -116,11 +121,6 @@ func TestFullClusterBackup(t *testing.T) {
 
 		// Pause SQL Stats compaction job to ensure the test is deterministic.
 		sqlDB.Exec(t, `PAUSE SCHEDULES SELECT id FROM [SHOW SCHEDULES FOR SQL STATISTICS]`)
-
-		// Disable automatic stats collection on the backup and restoring clusters to ensure
-		// the test is deterministic.
-		sqlDB.Exec(t, `SET CLUSTER SETTING sql.stats.automatic_collection.enabled=false`)
-		sqlDBRestore.Exec(t, `SET CLUSTER SETTING sql.stats.automatic_collection.enabled=false`)
 
 		// Create some other descriptors as well.
 		sqlDB.Exec(t, `

